documentation/versioning_policy.markdown

```diff
@@ -79,11 +79,11 @@ Commands can be versioned on an individual command basis, so we are generally fr
 
 Commands are primarily represented by a corresponding wire format. Wire formats are versioned along with the corresponding command.
 
-The reasons to trigger a new command version are more common, and in general if we aren’t sure it's easy enough to create another version anyway.
+Changes to an existing command version can be made if the change is largely deemed backward compatible (so long as the consumer is less strict about new unexpected parameters). For example, addition of optional parameters within a command or wire format will not require a new version. 
```

Comment:
Most of that wording was copied from the querying part of the versioning policy, but I can edit it.